### PR TITLE
Bump current opset to 19

### PR DESCRIPTION
**Pull request by p-wysocki**
Signed-off-by: p-wysocki <przemyslaw.wysocki@intel.com>

### Description
Release branch has been cut, new opset can be introduced. Based on https://github.com/onnx/onnx/pull/4134.

### CODE REVIEW

**Comment by gramalingam** [comment]
> Looks like the compiler throws an error due to an unused variable `fn`. Should I close the PR and leave adding this to the first new operator in opset19 or should I add a dummy placeholder?

I think you should be able to just have a dummy use, like `(void) fn;`  (if that's what you meant).

**Review by jcwchen — APPROVED** [review]
LGTM. I think this PR will be a better demo PR than mine (https://github.com/onnx/onnx/pull/4134), because it handles `(void)fn` :) Probably we can update  the demo PR in OnnxReleases.md.

**Comment by p-wysocki** [comment]
@jcwchen Done in https://github.com/onnx/onnx/pull/4683.

File: onnx/defs/operator_sets.h [hunk 1]
```diff
@@ -1058,6 +1058,16 @@ class OpSet_Onnx_ver18 {
   }
 };
 
+// Forward declarations for ai.onnx version 19
+
+// Iterate over schema from ai.onnx version 19
+class OpSet_Onnx_ver19 {
+ public:
+  static void ForEachSchema(std::function<void(OpSchema&&)> fn) {
+    (void)fn;
```
Comment (jcwchen) [inline onnx/defs/operator_sets.h:1067]: tiny nit: perhaps add a TODO saying this line should be removed after introducing an actual new schema?

Comment (p-wysocki) [reply onnx/defs/operator_sets.h:1068]: Done.

**Pull request merged**